src/exampleTypes/ResultTable.js

```diff
@@ -41,6 +41,84 @@ let popoverStyle = {
   userSelect: 'none',
 }
 
+let HighlightedColumn = withStateLens({ viewModal: false })(
+  observer(
+    ({
+      visibleFields,
+      node,
+      highlight = _.result('slice.0.highlight', getResults(node)),
+      header,
+      Cell = header ? 'th' : 'td',
+      Table = 'table',
+      Modal = null,
+      viewModal,
+    }) => {
+      let alsoHighlighted = _.difference(_.keys(highlight), visibleFields)
```

Comment:
We already solve this problem in the results type via `additionalFields` https://github.com/smartprocure/contexture-elasticsearch/blob/master/src/example-types/results.js#L73

Reply:
Does it though? I'm testing and if the highlight object includes a property that is not in the includes, I don't see it in additionalFields

Reply:
My chat with Sam:
> **Daniel:** so, what am I doing wrong? I'm removing "summary" from the includes, but I'm not getting "summary" on the additional fields
> **Sam:** Check the source of results, there might be a bug. Maybe we never tested it with includes? I mean that would be weird lol but maybe.

Reply:
https://github.com/smartprocure/contexture-elasticsearch/pull/95